_benchmark/metrics/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Metrics
+nav_order: 25
+has_children: true
+---
+
+# Metrics
+
+After a workload completes, OpenSearch Benchmark stores all metric records within its metrics store. These metrics can be kept in memory or inside of an OpenSearch cluster. 
+
+## Storing metrics
+
+You set where metrics are stored using the [`datastore.type`](https://opensearch.org/docs/latest/benchmark/configuring-benchmark/#results_publishing) parameter in your `benchmark.ini` file. 
+
+### In memory
+
+If you want to store your result in memory, use the following settings in the `results_publishing` section of `benchmark.ini`:
+
+```json
+[results_publishing]
+datastore.type = in-memory
+datastore.host = 
+datastore.port = 
+datastore.secure = False
+datastore.ssl.verification_mode = 
+datastore.user = 
+datastore.password =
+```
+
+### OpenSearch
+
+If you want to store your results in an external OpenSearch memory store, use the following settings in the `results_publishing` section of `benchmark.ini`:
+
+```
+[results_publishing]
+datastore.type = opensearch
+datastore.host = <opensearch endpoint>
+datastore.port = 443
+datastore.secure = True
+datastore.ssl.verification_mode = none
+datastore.user = <opensearch basic auth username>
+datastore.password = <opensearch basic auth password>
+```
+
+After you run OpenSearch Benchmark configured to use OpenSearch as a datastore, OpenSeach Benchmark creates a `benchmark-metrics` Dashboard, where you can find your metric results by , as shown in the following image.
```

Comment:
The OSB doesn't create any dashboard. OSB creates three indices, `benchmark-metrics-YYYY-MM` that holds granular metrics data and telemetry data, `benchmark-results-YYYY-MM` that holds the final results data and `benchmark-test-executions-YYYY-MM` that hold the data around execution-ids. 
The user can use OpenSearch Dashboards to visualize data on these indices.